packages/notifi-react-card/README.md

```diff
@@ -33,6 +34,21 @@ export const Remote: React.FC = () => {
     return null;
   }
 
+  const inputLabels = {
+    email: 'Email',
+    sms: 'Text Message',
+    telegram: 'Telegram',
+  };
+
+  const inputSeparators: NotifiInputSeparators = {
+    first: {
```

Comment:
hey! i think this should be 

`emailSeparator` and respectively `smsSeparator`!

Reply:
updated